guides/common/modules/proc_creating-a-webhook.adoc

```diff
@@ -0,0 +1,55 @@
+[id="creating-a-webhook_{context}"]
+= Creating a Webhook
+
+You can customize events, payloads, HTTP authentication, content type and headers through {ProjectWebUI}. 
+Use the following procedure to create a webhook in the {ProjectWebUI}.
+
+.Procedure
+
+. In the {ProjectWebUI}, click *Administer* -> *Webhooks*.
+. Click *Create Webhook*.
+. Click *Subscribe to field* to select an event.
+. Enter a name.
+. Enter a target URL.
```

Comment:
Not sure how to mention this exactly, but `Target URL` supports simple ERB in it. For example you can configure a "dynamic" URL:

`Target URL - https://server.example.com/api/hosts/<%= @object.id %>/play_roles`